docs/next.config.js

```diff
@@ -33,28 +32,4 @@ module.exports = withCSS({
     };
     return config;
   },
-  async exportPathMap(defaultPathMap, { dev, dir, outDir }) {
-    if (dev) {
-      return defaultPathMap;
-    }
-    copySync(join(dir, 'robots.txt'), join(outDir, 'robots.txt'));
-    return Object.assign(
-      ...Object.entries(defaultPathMap).map(([pathname, page]) => {
-        if (pathname.match(/\/v[1-9][^\/]*$/)) {
-          // ends in "/v<version>"
-          pathname += '/index.html'; // TODO: find out why we need to do this
-        }
-        if (pathname.match(/unversioned/)) {
-          return {};
-        } else {
-          // hide versions greater than the package.json version number
-          const versionMatch = pathname.match(/\/v(\d\d\.\d\.\d)\//);
-          if (versionMatch && versionMatch[1] && semver.gt(versionMatch[1], version)) {
-            return {};
-          }
-          return { [pathname]: page };
-        }
-      })
-    );
-  },
-});
```

Comment:
I'm concerned that changing this will break something. It is not sufficient to run the site and browse around. There are existing links on the internet that need to keep working. S3 is sensitive to trailing slashes.

I would make sure that links that work today will continue to work with S3. That means that if links with trailing slashes work, they should continue to work. And same for links without trailing slashes.

I would also compare the list of files generated before and after this commit.